jul-to-slf4j/src/main/java/org/slf4j/bridge/SLF4JBridgeHandler.java

```diff
@@ -300,4 +313,26 @@ public void publish(LogRecord record) {
         }
     }
 
+    /**
+     * A j.u.l. configuration class which registers {@link SLF4JBridgeHandler} as handler for the j.u.l. root logger.
+     * <p/>
+     * Intended to be used in conjunction with the <em>java.util.logging.config.class</em> system property.
+     *
+     * @author Mark Fulton
+     * @see java.util.logging.LogManager
+     */
+    public static class Initializer {
+
+        /**
+         * Registers {@link SLF4JBridgeHandler} as handler for the j.u.l. root logger.
+         *
+         * @throws IOException if there are problems reading from the stream.
+         */
+        public Initializer() throws IOException {
+            String loggingConfigurationString = "handlers = " + SLF4JBridgeHandler.class.getName();
```

Comment:
I've added a test locally and now I realize that I don't need to create an in-memory configuration file to read. I will push an update in a bit.